### DISCUSSION TITLE
fluxCD and  feature flags

### DISCUSSION THREAD

**Discussion by atmosx** [fluxcd/flux2#2146] [Q&A]
Hello!

I never used fluxCD or similar tools before. I looked around and right now fluxCD looks like the simplest and easiest tool to use.

There is no mention on how fluxCD implements feature flags in the [documentation](https://fluxcd.io/docs/).  Can someone share experiences, documentation or blog posts on using feature flags with fluxCD?

The use cases I'm interested in:

1) Application1 changes configuration, ideally, apply the configuration on a repository without having to rebuild the application.
2) We have to 8 clusters. We want to rollout `ingress-nginx` new version in clusters 1 and 2, without blocking the repository's flow.

Can fluxCD handle these cases without using a third party tool?

Thanks!

**Comment by tuxillo** [comment]
Did you find a solution for this?

**Reply by atmosx** [reply]
No. We did not move forward with FluxCD, so I stopped digging.
